examples/rllib_cylinder_ppo_tune.py

```diff
@@ -0,0 +1,164 @@
+import os
+
+os.environ["OMP_NUM_THREADS"] = "1"
+
+from ray import train, tune
+from ray.rllib.algorithms.ppo import PPOConfig
+
+from firedrake import logging
+import numpy as np
+from tap import Tap
+from typing import Iterable, Tuple, Union
+from numpy.typing import ArrayLike
+import gymnasium as gym
+
+import hydrogym
+from hydrogym.core import PDEBase, TransientSolver, CallbackBase
+
+logging.set_log_level(logging.DEBUG)
+
+
+class ArgumentParser(Tap):
+    reynolds_number: int = 100  # Reynolds number
+    mesh_resolution: str = "medium"  # Mesh resolution
+    time_step: float = 1e-2  # Time step
+    batch_size_per_learner: int = 2000  # Batch size per learner
+    number_of_epochs: int = 10  # Number of epochs
+    num_runners: int = 16  # Number of environment runners
+
+
+# Read in the command-line arguments
+args = ArgumentParser().parse_args()
+
+
+class FlowEnv(gym.Env):
```

Comment:
Needs to use the abstraction from the HydroGym core itself, and not monkey-patch itself.